### ISSUE TITLE
[Accessibility bug] The page does not have a lang= attribute will cause readability issue

### ISSUE THREAD

**Issue by seki2020** [topcoder-platform/Training-Challenges#66]
**Bug Title:** 
[Accessibility bug ] The page does not have a lang= attribute will cause readability issue

**Steps to Reproduce:**
Go to [https://connect.topcoder.com/projects/](https://connect.topcoder.com/projects)
Open Dev Console, see <html> tags of source.

**Actual Result:**
The <html> tag has no attribute, especially the **lang** attribute.

**Expected Result:** 
Put on lang="en" to it, and it will obviously increase readability in Accessibility wise.
Ref material:
[https://www.w3.org/TR/WCAG21/#language-of-page](https://www.w3.org/TR/WCAG21/#language-of-page)

**Screenshots or screencast URL:**
![image](https://user-images.githubusercontent.com/46511237/72602856-ec226500-3952-11ea-8c25-d74c8bce0827.png)

**Device/OS/Browser Information:**
On PC and macOS from both Chrome and Safari
